lints/0003_auth_rls_initplan.sql

```diff
@@ -15,8 +15,17 @@ with policies as (
             when 'd' then 'DELETE'
             when '*' then 'ALL'
         end as command,
-        qual,
-        with_check
+        -- normalize expression's spaces
+        regexp_replace(
+            regexp_replace(
+                regexp_replace(qual, '[\n\r\t ]+', ' ', 'g'),
+            ' *\( \)', '()', 'g'),
+        ' *\. *', '.', 'g') as qual,
+        regexp_replace(
+            regexp_replace(
+                regexp_replace(with_check, '[\n\r\t ]+', ' ', 'g'),
+            ' *\( \)', '()', 'g'),
+        ' *\. *', '.', 'g') as with_check
```

Comment:
when policies get written to pg_policy they get normalized automatically. have you seen whitespace issues come up?

Reply:
confirmed its auto-normalized